source/ranges.tex

```diff
@@ -3202,6 +3210,429 @@
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\expects
+If \tcode{Bound} is not \tcode{unreachable_sentinel_t},
+$\tcode{bound} \ge 0$.
+
+\pnum
+\effects
+Initializes \exposid{value_} with \tcode{value} and
+\exposid{bound_} with \tcode{bound}.
+\end{itemdescr}
+
+\indexlibraryctor{repeat_view}%
+\begin{itemdecl}
+constexpr explicit repeat_view(W&& value, Bound bound = Bound());
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\expects
+If \tcode{Bound} is not \tcode{unreachable_sentinel_t}, $\tcode{bound} \ge 0$.
+
+\pnum
+\effects
+Initializes \exposid{value_} with \tcode{std::move(value)} and
+\exposid{bound_} with \tcode{bound}.
+\end{itemdescr}
+
+\indexlibraryctor{repeat_view}%
+\begin{itemdecl}
+template<class... WArgs, class... BoundArgs>
+  requires @\libconcept{constructible_from}@<W, WArgs...> &&
+           @\libconcept{constructible_from}@<Bound, BoundArgs...>
+constexpr explicit repeat_view(piecewise_construct_t,
+  tuple<Wargs...> value_args, tuple<BoundArgs...> bound_args = tuple<>{});
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Initializes \exposid{value_} with arguments of types \exposid{WArgs...}
+obtained by forwarding the elements of \tcode{value_args} and
+initializes \exposid{bound_} with arguments of types \tcode{BoundArgs...}
+obtained by forwarding the elements of \tcode{bound_args}.
+(Here, forwarding an element \tcode{x} of type \tcode{U} within a tuple object
+means calling \tcode{std::forward<U>(x)}.)
+\end{itemdescr}
+
+\indexlibrarymember{begin}{repeat_view}%
+\begin{itemdecl}
+constexpr @\exposid{iterator}@ begin() const;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to: \tcode{return \exposid{iterator}(addressof(*value_));}
+\end{itemdescr}
+
+\indexlibrarymember{end}{repeat_view}%
+\begin{itemdecl}
+constexpr @\exposid{iterator}@ end() const requires (!@\libconcept{same_as}@<Bound, unreachable_sentinel_t>);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to: \tcode{return \exposid{iterator}(addressof(*\exposid{value_}), \exposid{bound_});}
+\end{itemdescr}
+
+\indexlibrarymember{end}{repeat_view}%
+\begin{itemdecl}
+constexpr unreachable_sentinel_t end() const noexcept;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to: \tcode{return unreachable_sentinel;}
+\end{itemdescr}
+
+\indexlibrarymember{size}{repeat_view}%
+\begin{itemdecl}
+constexpr auto size() const requires (!same_as<Bound, unreachable_sentinel_t>);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to: \tcode{return \exposid{to-unsigned-like}(\exposid{bound_});}
+\end{itemdescr}
+
+\rSec3[range.repeat.iterator]{Class template \tcode{repeat_view::\exposid{iterator}}}
+
+\begin{codeblock}
+namespace std::ranges {
+  template<@\libconcept{copy_constructible}@ W, @\libconcept{semiregular}@ Bound = unreachable_sentinel_t>
+    requires @\exposid{is-integer-like}@<Bound> || @\libconcept{same_as}@<Bound, unreachable_sentinel_t>
```

Comment:
Mind https://github.com/cplusplus/draft/pull/5709#discussion_r939174850.

Reply:
@JohelEGP Could you say what exactly needs changing here?

Reply:
The fix has been applied. This was meant to call attention to the linked comment which had been outdated by an unrelated change.

Reply:
Aha, excellent -- then I think we're ready here!